src/debug-generator/debug-generator.c

```diff
@@ -84,14 +95,11 @@ static int parse_breakpoint_from_string(const char *s, uint32_t *ret_breakpoints
 
         /* Empty value? set default breakpoint */
         if (isempty(s)) {
-                if (in_initrd()) {
-                        FOREACH_ARRAY(i, breakpoint_info_table, ELEMENTSOF(breakpoint_info_table))
-                                if (i->validity & BREAKPOINT_DEFAULT) {
-                                        breakpoints |= 1 << i->type;
-                                        break;
-                                }
-                } else
-                        log_warning("No default breakpoint defined on the host, ignoring breakpoint request from kernel command line.");
+                FOREACH_ARRAY(i, breakpoint_info_table, ELEMENTSOF(breakpoint_info_table))
+                        if (FLAGS_SET(i->validity, BREAKPOINT_DEFAULT) && breakpoint_applies(i, LOG_WARNING)) {
```

Comment:
here you should use LOG_DEBUG or maybe even LOG_DEBUG+1 or so, because while searching we don't want any log messages to be shown

Reply:
or maybe INT_MAX is nicer than LOG_DEBUG+1, and has the same effect of suppressing the log message

Reply:
done, thanks